region/maridia/inner-yellow/Watering Hole.json

```
@@ -142,6 +142,57 @@
         "Requires a pixel-perfect freeze in order to run onto and off of the frozen Zebs."
       ]
     },
+    {
+      "link": [1, 1],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit) - Frozen Zeb",
```

We are planning on removing the gain blue / flash suit in strat titles soon. The first part of the name makes it clear, and too long of strat names causes problems with the spoiler log
```suggestion
      "name": "R-Mode Spark Interrupt - Frozen Zeb Runway",
```